### PR TITLE
Feat/add login screen

### PR DESCRIPTION
**Pull request by ChinyangaTL**
## Change Log
- Added and linked the login screen
- Created the login form 

Screenshot 2024-07-12 at 10 45 10

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <h1>Home</h1>
+    <a href="/login.html">to login</a>
```
Comment (LordNtlaloe) [inline index.html]: Capitalize the words "to login"

File: login.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Login</title>
+  </head>
+  <body>
+    <p>login</p>
+    <input type="text" placeholder="username" />
+    <input type="password" placeholder="password" />
+    <button>login</button>
```
Comment (LordNtlaloe) [inline login.html]: Capitalize the word "login"

File: index.html [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
     <title>StarsStruck</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <h1>Home</h1>
```
Comment (UniBelle) [inline index.html:9]: Capitalize Home

**Review by ragmer09toske — CHANGES_REQUESTED** [review]
hey, add some some UI

**Review by LordNtlaloe — CHANGES_REQUESTED** [review]

**Review by UniBelle — CHANGES_REQUESTED** [review]

**Review by UniBelle — APPROVED** [review]

**Review by ragmer09toske — APPROVED** [review]

**Review by LordNtlaloe — APPROVED** [review]

**Pull request merged**